lib/app/app.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:freecodecamp/ui/views/browser/browser_view.dart';
+import 'package:freecodecamp/ui/views/home/home_view.dart';
+
+import '../ui/views/startup/startup_view.dart';
+import '../ui/views/website/website_view.dart';
+import 'package:stacked/stacked_annotations.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+// Run 'flutter pub run build_runner build --delete-conflicting-outputs' after any changes in this file to generate updated files
```

Comment:
Hey @nikunjdk,
How exactly did you generate the code? When I use `flutter pub run build_runner build`  nothing seems to happen.

Reply:
You use it when you need to add new routes or services. It then generates apppp.locator and app.route files